website/docs/r/rde_type_behavior.html.markdown

```diff
@@ -49,22 +49,88 @@ resource "vcd_rde_type_behavior" "my_rde_type_behavior" {
+    "_internal_key" : "secretKey",
+    "execution_properties" : {
+      "template" : {
+        "content" : "<template_content_string>"
+      },
+      "_secure_token" : "secureToken",
+      "invocation_timeout" : 7
+    }
+  })
+}
+
+resource "vcd_rde_type" "my_rde_type" {
+  vendor        = "vmware"
+  nss           = "vcd"
+  version       = "4.5.6"
+  name          = "My VMware RDE Type"
+  interface_ids = [data.vcd_rde_interface.my_interface.id]
+  schema        = file("${path.module}/schemas/my-type-schema.json")
+
+  # Behaviors can't be created after the RDE Interface is used by a RDE Type
+  # so we need to depend on the Behavior to wait for it to be created first.
+  depends_on = [vcd_rde_interface_behavior.my_interface_behavior]
+}
+
+resource "vcd_rde_type_behavior" "my_rde_type_behavior" {
+  rde_type_id               = vcd_rde_type.my_rde_type.id
+  rde_interface_behavior_id = vcd_rde_interface_behavior.my_interface_behavior.id
+  execution_json = jsonencode({
+    "type" : "WebHook",
+    "id" : "testWebHookOverridden",
+    "href" : "https://hooks.slack.com:443/services/T07UZFN0N/B01EW5NC42D/rfjhHCGIwzuzQFrpPZiuLkIX",
+    "_internal_key" : "secretKey2",
+    "execution_properties" : {
+      "template" : {
+        "content" : "<template_content_string>"
+      },
+      "_secure_token" : "secureToken",
+      "invocation_timeout" : 7
+    }
+  })
+}
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
 * `rde_type_id` - (Required) The ID of the RDE Type that owns the Behavior
 * `rde_interface_behavior_id` - (Required) The ID of the [RDE Interface Behavior](/providers/vmware/vcd/latest/docs/resources/rde_interface_behavior) to override
 * `description` - (Optional) The description of the RDE Type Behavior.
-* `execution` - (Required) A map that specifies the Behavior execution mechanism.
+* `execution_json` - (Optional; *v3.14*) A string representing a valid JSON that specifies the Behavior execution mechanism.
   You can find more information about the different execution types, like `WebHook`, `noop`, `Activity`, `MQTT`, `VRO`, `AWSLambdaFaaS`
-  and others [in the Extensibility SDK documentation](https://vmware.github.io/vcd-ext-sdk/docs/defined_entities_api/behaviors)
+  and others [in the Extensibility SDK documentation](https://vmware.github.io/vcd-ext-sdk/docs/defined_entities_api/behaviors).
+  One of `execution_json` or `execution` must be set.
+* `always_update_secure_execution_properties` - (Optional; *v3.14*) Useful to update execution properties marked with `_secure_` and `_internal_`
+  as these are not retrievable from VCD, so they are not saved in state. Setting this to `true` will make the provider
+  to ask for updates whenever there is a secure property in the execution of the Behavior
+* `execution` - (Optional) A map that specifies the Behavior execution mechanism, this is just a simplification of `execution_json` that
```

Comment:
So, my overall question is why we remove `execution` way from test and example, if we still leave it available.

Reply:
I think both cases is diff view messing around.